go.mod

```diff
@@ -10,12 +10,10 @@ require (
 	github.com/gagliardetto/treeout v0.1.4
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.12.1
-	github.com/satori/go.uuid v1.2.0
-	github.com/smartcontractkit/chainlink v1.3.0
+	github.com/smartcontractkit/chainlink v1.4.1-rc1.0.20220512210926-5363811660bf
```

Comment:
Is this a stable commit on develop or a branch? (just concerned that a squash may leave this commit to be garbage collected by github sometime in the future)

Reply:
will need to do a follow up, since this is an interface change its a 3 PR process